src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected char type;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        type = 'T';
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    @Override
+    public String toString() {
+        return "[" + type + "][" + getStatusIcon() + "]" + description;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void markNotDone() {
+        isDone = false;
```

Comment:
May want to include some code to check if the user is attempting to markAsDone an already done task, and same for markNotDone, and respond to user accordingly.